front/lib/api/assistant/agent.ts

```diff
@@ -297,27 +299,33 @@ async function* runMultiActionsAgentLoop(
   }
 }
 
-async function getEmulatedAgentMessageActions(
+async function getEmulatedAndJITActions(
```

Comment:
I'd rather separate emulated and JIT like I did in my PR, in part for the reason below and because they are treated quite differently (steps etc..)